src/main/java/io/debezium/connector/yugabytedb/YugabyteDBConnectorConfig.java

```diff
@@ -1433,6 +1529,161 @@ private static int validateLogicalDecodingMessageExcludeList(Configuration confi
         return 0;
     }
 
+    public static String extractStreamIdFromSlot(Configuration config) {
+        RuntimeException exception = null;
+        try (YugabyteDBConnection ybConnection = new YugabyteDBConnection(getJdbcConfig(config),
+                YugabyteDBConnection.CONNECTION_GENERAL)) {
+            int retryCount = 0;
+            while (retryCount <= config.getInteger(MAX_RETRIES)) {
+                try (Connection connection = ybConnection.connection();
+                      final Statement statement = connection.createStatement()) {
+                    String query = "SELECT yb_stream_id FROM pg_replication_slots WHERE slot_name = '"+ config.getString(SLOT_NAME) + "';";
+                    final ResultSet rs = statement.executeQuery(query);
+                    if (rs.next()) {
+                        String streamId = rs.getString("yb_stream_id");
+                        return streamId;
+                    } else {
+                        LOGGER.error("StreamId not found");
```

Comment:
Add more detail in the error log i.e. Stream ID not found in the replication slots.

Reply:
This will happen if the replication slot with the given name doesn't exist. That is the only case, when there will be an empty result set.

So let's log the error that `Could not find the replication slot with name '__name__'`